docs/specification.md

```diff
@@ -569,6 +517,65 @@ Parameter (in Michelson):
 - Since the contract supports only a single token type, `tokenId` must be 0.
   It is passed because FA2 requires that.
 
+**Custom (non-FA2) token functions**
+====================================
+
+Functions for the stablecoin token implementation not present in FA2, but related to token transfers.
+
+**approveOperatorUpdate (TODO)**
```

Comment:
Also we should fix its type to be a unit since we don't bother setting operators for multiple token types

Reply:
`unit`? Type of `approveOperatorUpdate`? What will it do then?

Reply:
Once is called sender will be defined in transfer's ledger value as operator

Reply:
Removed this function as explained in the last commit.